DEVELOPER-SETUP.md

```diff
@@ -87,6 +88,7 @@ npm install
 ```sh
 mkdir assets/romm
 ln -s ../../../romm_mock/resources assets/romm/resources
+ln -s ../../../romm_mock/assets assets/romm/assets
```

Comment:
So to keep a very distinct line between stuff uploaded by users, and stuff imported from igdb, i added a new volume `assets`. It's more likely users will want to bind this access them directly, then the igdb resources.